pkg/types/vsphere/conversion/installconfig.go

```diff
@@ -21,11 +27,59 @@ const (
 	GeneratedFailureDomainZone string = "generated-zone"
 )
 
-// ConvertInstallConfig modifies a given platform spec for the new requirements.
-func ConvertInstallConfig(config *types.InstallConfig) error {
-	platform := config.Platform.VSphere
+// GetFinder connects to vCenter via SOAP and returns the Finder object if the SOAP
+// connection is successful. If the connection fails it returns nil.
+// Errors are mostly ignored to support AI and agent installers.
+func GetFinder(server, username, password string) (*find.Finder, error) {
```

Comment:
@rvanderp3 I plan on removing the auth part of this for 4.16 once CAPV changes have merged.